packages/create-plugin/templates/provisioning-template/provisioning/datasources/datasources.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: 1
+deleteDatasources:
+  - name: TestData DB
+    orgId: 1
+datasources:  
```

Comment:
Could we add an example here that contains jsonData and secureJsonData? Even better, add a link to the documentation about provisioning datasources.